Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml

```diff
@@ -125,6 +125,6 @@ tags:
 - phishing
 timeout: '0'
 type: python
-dockerimage: demisto/dl:1.3
+dockerimage: demisto/dl:languages1.0
```

Comment:
@lperryp / @erezh31 is this ok?

Reply:
It's better to leave demisto/dl:1.3, the newer one is larger